### ISSUE TITLE
How to ensure toots from other time zones or sent at unusual times are visible in crowded timelines?

### ISSUE THREAD

**Issue by jukey** [mastodon/mastodon#23282]
### Pitch

Having a crowded timeline is great but you cannot read everything posted anymore. There are always fresh toots and they are all ordered chronological. The downside is that it becomes hard to follow toots that are sent at completely different times. There is no good way to see toots that are send by users in europe, asia and amercia. This issue is about discussing potential solutions to this problem.

### Motivation

In order to see toots from a different timezone one could follow one of the existing proposals:
- implement a randomize timeline feature as proposed in https://github.com/mastodon/mastodon/issues/21209
- show tweets from all accounts one follows equally distributed not preferring the more active ones as proposed in https://github.com/mastodon/mastodon/issues/18352 
- show toots from accounts one follows based on toot popularity as proposed in https://github.com/mastodon/mastodon/issues/15981 and https://github.com/mastodon/mastodon/issues/3782

Other ideas that I did not found in the issues section but maybe worth following could be:
- allow users to use an offset: show toots 3/6/12 hours ago
- allow users to select a geographical area: show toots from europe/india/mexico/etc.
- allow users to pick a time zone

If this is not the right place to discuss such ideas please feel free to point me to a better fitting one.

**Comment by Ciantic** [comment]
Yes, I agree this is a good idea.

It's currently impossible to get older toots from the home timeline. It is limited to 400 toots: https://github.com/mastodon/mastodon/issues/20173 For me, this is ~3-5 hours. 

That needs to be fixed before we can even build a third-party tool to unearth older toots. 

Takahe, alternate implementation, on the other hand, can store an infinite amount, currently, they planned ~30 days.

**Comment by TimothyRichardGreen** [comment]
Similarly, a couple of people I follow have the habit of posting in bursts. 10 to 20 posts in a row, once or twice a week. If I'm active when they post, I see little else. If I'm not active when they post, I don't see them at all. Some sort of shuffle would be nice.
